### PR TITLE
docs(behavior_path_planner): update avoidance module documentation

### PR DESCRIPTION
**Pull request by zulfaqar-azmi-t4**
Signed-off-by: Muhammad Zulfaqar Azmi <zulfaqar.azmi@tier4.jp>

## Description

Recently, there are updates (#1055 and #1041)  with regards to generating shift points in `avoidance_module`'.
This documentation will update documentation related to them.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: planning/behavior_path_planner/behavior_path_planner_avoidance-design.md [hunk 1]
```diff
@@ -261,24 +259,11 @@ To compute the shift length, in addition to the vehicle's width and the paramete
   - It is recommended to set the value to more than half of the ego vehicle's width.
 - The `road_shoulder_safety_margin` will prevent the module from generating a path that might cause the vehicle to go too near the road shoulder.
 
-![fig1](./image/shift_length_parameters.png)
-
-The shift length is subjected to the following constraints.
+![shift_length_parameters](./image/shift_length_parameters.drawio.svg)
+q
```
Comment (TakaHoribe) [inline planning/behavior_path_planner/behavior_path_planner_avoidance-design.md:263]: "q" Typo?

Comment (zulfaqar-azmi-t4) [reply planning/behavior_path_planner/behavior_path_planner_avoidance-design.md]: Sorry! It is removed already.

Comment (TakaHoribe) [reply planning/behavior_path_planner/behavior_path_planner_avoidance-design.md]: :+1:

**Comment by zulfaqar-azmi-t4** [comment]
@taikitanaka3 
I will not yet merge the PR, since there is some limitation that should be addressed, which is the shift point unexpected behavior when the size of the obstacle bounding box changes unexpectedly. Therefore, I will need to make another commit.

**Pull request merged**